### PR TITLE
FizzBuzz Python challenge

### PR DESCRIPTION
**Pull request by MishManners**
I wanted to add this in as I had so much trouble with building this function and couldn't find a lot of useful help online. Hope this helps people out!

### CODE REVIEW

File: FizzBuzz.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        print ('fizz')
+    elif n % 5 == 0:
+        print ('buzz')
+    else:
+        print(n)
+
+# input function
+for input in inputs:
+    input = int(input)
+    fizzbuzz(input)
```
Comment (thevirtualbuddy) [inline FizzBuzz.md:34]: `function_name(val) # expected result`
could be added at the end of the file.

**Review by thevirtualbuddy — COMMENTED** [review]
`function_name(val) # result`
could be added at the end of the file. 
That's would follow the snippet-template file then.

**Comment by Melvin-Abraham** [comment]
Couple of things to mention as per `CONTRIBUTING.md`:

- Filename should be specified as `fizzbuzz.md` as per the function's name instead of `FizzBuzz.md`. Following the same convention, the `title` should also be `fizzbuzz`.

  > Snippet titles must correspond to the filename and follow the language and repository's naming conventions

- You seem to have removed `snippet-template.md` which is something that shouldn't be edited/removed as it is used by `integration-tools` ([refer here](https://github.com/30-seconds/30-seconds-of-python/blob/master/CONTRIBUTING.md#tools))

  > **Only modify snippet files** ...

- The tag `practice` specified in the snippet doesn't reflect any valid tags on the website. ([website link](https://www.30secondsofcode.org/python/p/1))

  > Snippet tags must be comma-separated, contain a **primary tag as seen on the website** as their first tag

**Review by Trinityyi — COMMENTED** [review]
Thanks for contributing. However, your PR doesn't follow our [contribution guidelines](../blob/master/CONTRIBUTING.md) correctly and it doesn't provide enough value for the average reader. Closing due to that.

**Comment by MishManners** [comment]
Oh yes I'm so sorry. Let me get this fixed up and re-sent. Thank you.

**Comment by MishManners** [comment]
I also am unsure what happened to the snippet... it it still there in the file...

**Pull request closed without merging**